### PR TITLE
Add InfluxDB support for datafeeds.

### PR DESCRIPTION
**Pull request by rterbush**
This is a good start. Probably a few issues still to fix but appears to run a backtest at this point.

### CODE REVIEW

**Comment by rterbush** [comment]
@mementum what do I need to do to satisfy the missing influxdb module. Is this something I can use the new packages functionality to accomplish?

**Comment by mementum** [comment]
You need to buy the project some time, which is always a scarce resource.

Will be tackled soon

**Comment by rterbush** [comment]
Happy to provide some of my time if that will help.

**Comment by mementum** [comment]
Already provided

**Comment by mementum** [comment]
The support should be complete (probably was enough for this implementation). Will be pulled later today and integrated

**Comment by rterbush** [comment]
@mementum one heads up here. For some reason, I am not able to get daily data from this feed after recent updates. I see the bars in correct timeframe in `_load():`, but no daily bars are available in the strategy for backtest. I'll dig into this a bit more today.

File: backtrader/feeds/influxfeed.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                'FROM "{dataname}" '
+                'WHERE time {begin} '
+                'GROUP BY time({timeframe}) fill(none)').format(
+                    open_f=self.p.open, high_f=self.p.high,
+                    low_f=self.p.low, close_f=self.p.close,
+                    vol_f=self.p.volume, oi_f=self.p.ointerest,
+                    timeframe=tf, begin=st, dataname=self.p.dataname)
+
+        try:
+            dbars = list(self.ndb.query(qstr).get_points())
```
Comment (josefstr) [inline backtrader/feeds/influxfeed.py]: dbars has length of 10000
so you're limited with loading 10000 rows?

Comment (rterbush) [reply backtrader/feeds/influxfeed.py]: @josefstr I seem to remember this is a limitation controlled by your influxd configuration.

Comment (josefstr) [reply backtrader/feeds/influxfeed.py]: so you can load more than 10000 rows? I cannot find any limitations in the influxd configuration, could you give me a hint? Thanks :)

Comment (josefstr) [reply backtrader/feeds/influxfeed.py]: there are 3 issues:
https://github.com/influxdata/influxdb/issues/8222
https://github.com/influxdata/influxdb-python/issues/450
https://github.com/influxdata/influxdb-python/issues/463

Looking at a tick time series I have more than 20 million rows per symbol, so it's not possible to load them all at once and then process.
I'm not familiar with python, but we need to iterate over the result somehow and process the incoming chunks sequentially like in a stream.

Comment (rterbush) [reply backtrader/feeds/influxfeed.py]: As you point out, this appears to be an issue in influxdb-python and not backtrader.
There are other options for storing and querying time series data. You might want to look at timescale.com as an option for your larger datasets.

This should probably be discussed in the community thread.

Comment (josefstr) [reply backtrader/feeds/influxfeed.py]: I'd like to open a issue on this, but they are disabled on github for this repo?

Comment (rterbush) [reply backtrader/feeds/influxfeed.py]: @josefstr the issue is not with the code in this project but instead is an issue/limitation of the python module that is being used. If you can find another python module that has fixed this limitation, or convince the team responsible for the influxdb-python code to fix their issue, we should be able to implement those changes.

**Pull request merged**